Add Expectations for Beatmap Nominators (#4281)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/Expectations/en.md

# Expectations for Beatmap Nominators

*See also: [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules)*

> As central figures in their respective communities and osu! as a whole, [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) are held to a higher standard than other users. The expectations listed here are the result of discussion within the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). While it's unreasonable to perfectly align with these expectations, going far out of line can lead to warnings and/or removal from the Beatmap Nominators. All situations are judged on a case-by-case basis depending on frequency, severity, judgement, and impact.

Reviewer: > judged on a case-by-case basis depending on frequency, severity, judgement, and impact.

not sure what "judgement" is referring to here and it doesn't seem necessary

Author: hm, might not be the best word then
wanted a summarizing way of referring to the difference between a case being obvious (applying common sense) and a case where someone encounters a more difficult situation and makes a poor, but understandable, decision
having written that out, "obviousness" or even "common sense" could be a suitable replacement then?

As a reminder, Beatmap Nominators are expected to do their best to adhere to the [code of conduct for modding and mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) at all times. They should also always act within the best interest of maintaining the ranked section's quality.

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Reviewer: ```suggestion
- **The Golden Rule: Do not fuck up.** Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.
```
so the bold part actually says something 🤔 

also I know it's meant to have the casual-but-angery tone but you could probably think of something more representative than "do not fuck up". maybe the "act with sound judgement in all circumstances" from end of this sentence?

Reviewer 2: i would normally agree but "do not fuck up" has not had any sort of equivalent replacement that adequately encompasses all that it covers, and believe me, we've tried to figure out a different way to word it!

the unspoken onus of "not fucking up" is so powerful a thing that it's been part of the internal team ethos for like 8+ years, open to suggestions on how to reword it though

Reviewer 2: (peppy mentioned this below in [this comment](https://github.com/ppy/osu-wiki/pull/4281#discussion_r498433487) so i guess we're addressing it)

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Reviewer 2: we've had this for 8+ years and the definition of "fucking up" has only ever included making mistakes repeatedly and not learning from them, though i can see how it might be taken to "avoid failure at all costs instead of learning from it"

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Reviewer 2: ```suggestion
- **The Golden Rule: do not fuck up**. Fucking up in this instance is defined as a catch-all term referring to behavior like repeatedly making mistakes and never learning from them, acting rashly without regard for others, not approaching delicate situations with the caution they require and other examples of doing before thinking or not acting with the community's best interest in mind.
```

something like this perhaps, or do you want to omit the 'fuck up' thing entirely?

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Reviewer 3: i really dislike having this kind of language in "official" rules.

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Reviewer 2: ```suggestion
- **Act with the community's best interests at heart.** Approach tenuous situations with caution and care, aim to act with sound judgment wherever possible, seek to learn from your mistakes and at all times, make sure you're thinking about the impact your actions might have on the broader community.
```

something like this?

> - **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.

Author: Alternatively, "Act with forethought" encompasses the expectation well. Though Ephemeral's line has the special kind of cheesy-ness that fits the mindset behind these expectations well

> - **Act with the community's best interests at heart.** Approach tenuous situations with caution and care, aim to act with sound judgment wherever possible, seek to learn from your mistakes and at all times, make sure you're thinking about the impact your actions might have on the broader community.

Reviewer 4: ```suggestion
- **Act with the community's best interests at heart.** Approach tenuous situations with caution and care, aim to act with sound judgment wherever possible, seek to learn from your mistakes, and at all times make sure you're thinking about the impact your actions might have on the broader community.
```
Consistency with Oxford comma

- **Do not exploit or abuse existing systems.** Following the [Ranking Criteria](/wiki/Ranking_Criteria) and not trying to circumvent it is a given, and cases of disagreement should be resolved through a Ranking Criteria proposal. Beatmap Nominators should also not repeatedly exploit bugs or reward systems such as [score](/wiki/Score), [PP](/wiki/Performance_points) or [kudosu](/wiki/Modding/Kudosu).

> - **Be civil outside the game's direct platform.** Beatmap Nominators are expected to not be openly antagonistic, inflammatory, or otherwise derogatory toward other members of the osu! community, both in and outside of the game's direct platform. This especially applies to large public/semi-public communities where other users commonly associate them to their account on osu! (e.g. mapping Discord servers or Twitter).

Reviewer: idk if this is the time to comment on the rules themselves but this one seems open to abuse in the opposite direction. especially with such lax terms like "antagonistic" and "derogatory", it can be taken to mean "don't call ppl out on their BS in public" which also translates to "let the ppl in charge handle it privately" which also translates to not even close to what people are hoping to happen sometimes.

of course there's plenty of less thoughtful antagonizing etc. going on too, I just think this wording covers way too much

Reviewer 2: would swapping 'openly' to 'excessively' achieve a more reasonable end here?

Reviewer: seems better if that's the intention, yeah

> - **Be civil outside the game's direct platform.** Beatmap Nominators are expected to not be openly antagonistic, inflammatory, or otherwise derogatory toward other members of the osu! community, both in and outside of the game's direct platform. This especially applies to large public/semi-public communities where other users commonly associate them to their account on osu! (e.g. mapping Discord servers or Twitter).

Reviewer: ```suggestion
- **Be civil outside the game's direct platform.** Beatmap Nominators are expected to not be openly antagonistic, inflammatory, or otherwise derogatory toward other members of the osu! community, both in and outside of the game's direct platform. This especially applies to communities where other users commonly associate them to their account on osu! (e.g. mapping Discord servers or Twitter).
```

- **Mod regularly.** Beatmap Nominators are not only responsible for fixing unrankables, but also making the mapsets they're interested in nominating ready for rank, and ensuring their quality. They should always be vigilant and ready to mod thoroughly, just like they would have for their BN application.
- **Be approachable.** As members of the community who are often sought after, Beatmap Nominators should avoid giving the impression that they're unreachable or unwilling to communicate. However, this does not mean Beatmap Nominators are obliged to accept all requests.
- **Promote variety.** As main contributors to the ranked section, Beatmap Nominators should aim to provide the community with content from different mappers. They can have preferences, but their nominations should not only consist of the same mappers they, or to a lesser extent other BNs, have already nominated recently.
- **Keep improving!** Becoming a Beatmap Nominator is only the start of one's journey. There's still much to discuss and learn about. Beatmap Nominators are not only expected to help others, but also to ask for help when they're unsure about something.

> Those wishing to become a Beatmap Nominator should understand these expectations before seeking to join.

Reviewer: can prob link "seeking to join" to the becoming nominator article
